### PR TITLE
Allows Vox to hold limited positions on Torch

### PR DESCRIPTION
**Pull request by EgorDinamit**
## About the Pull Request

- Vox can now hold several contractor-only positions on Torch.
- Vox survival boxes can now be affected by the "premium boxes" station trait.
- Vox now start with sandals on and emergency tank on the belt, instead of in hand.
- Added 2 Vox clothes loadout options.

## Why It's Good For The Game

- Vox ship is rather rare and this change was discussed with the Vox maintainer. For more info look in Discord.
- Makes sense for it to be affected by the trait that affects every other survival box.
- Vox can't wear normal footwear, just like Unathi. Station Vox would spawn with a tank in-hand and drop it due to being in cryo, so it is fixed.
- Fancy Vox clothes, if someone wants to choose them.

## Did you test it?

Yes, all works.

vox-start

## Authorship

Me.

## Changelog

:cl:
tweak: Vox species can now work on SEV Torch. Their positions are heavily limited to contractor ranks only. They can also be a merchant now.
tweak: Added 2 Vox clothes options to loadouts.
tweak: Vox spawn with sandals on now.
/:cl:

### CODE REVIEW

File: code/modules/species/outsider/vox.dm
Comment (Shadowtail117) [inline code/modules/species/outsider/vox.dm:129]: Why make it an emergency tank instead of a regular one? It's a bit silly to suggest that Vox are constantly in a state of "emergency" breathing.

**Review by Shadowtail117 — CHANGES_REQUESTED** [review]
Changelog is a bit silly, this is a ship, not a station.

File: code/modules/species/outsider/vox.dm
Comment (EgorDinamit) [reply code/modules/species/outsider/vox.dm:129]: The issue is that you can't hold a regular tank in any slot, except for the back.
This is especially an issue when you have a backpack(on the ship/station), because if the tank is placed in your hand - you just instantly drop it due to being put in cryogenic storage.

Comment (EgorDinamit) [reply code/modules/species/outsider/vox.dm:129]: And if this matters, Vox already filters air with their gasmasks, so the tank isn't really needed.

Comment (Shadowtail117) [reply code/modules/species/outsider/vox.dm:129]: If this is the case, I think it is worth considering making tanks equippable on the belt slot. I know Vox are not looked on favorably but I don't think it's a good look to give them tanks reserved for emergency decompression. @Cupax3 thoughts?

Comment (Shadowtail117) [reply code/modules/species/outsider/vox.dm:129]: > And if this matters, Vox already filters air with their gasmasks, so the tank isn't really needed.

Oh. Are you certain the partial pressure is good? If so, the tank isn't really needed at all.

Comment (EgorDinamit) [reply code/modules/species/outsider/vox.dm:129]: Walked around with a mask on, no tank - didn't suffocate.
I want to keep the tank in mainly for the fact Vox on off-ship spawn on a decompressed(?) map, as far as I know.

Comment (Shadowtail117) [reply code/modules/species/outsider/vox.dm:129]: Why not make those spawns have the tanks while Torch spawns don't?

This isn't a huge point of contention, but if it can be optimized before merging it'd be ideal.

Comment (EgorDinamit) [reply code/modules/species/outsider/vox.dm:129]: I am not entirely sure if it's worth adding a lot of checks for the type of the spawn only to make sure one spawns with a tank and the other doesnt.
I honestly think it's better to just remove it entirely and make atmosphere on Vox off-ship pure nitrogen.

Comment (Snuggley) [reply code/modules/species/outsider/vox.dm:129]: I don't believe the Vox Offship is supposed to start decompressed, but it most certainly has on me in the past. I'm in favor of keeping the tanks for that reason, but also don't see it as a major issue considering the state of the offship in general.

**Comment by Snuggley** [comment]
This all looks good to me. The tanks are a minor issue since Vox Offship doesn't really spawn atm anyway.

**Comment by Shadowtail117** [comment]
CL still needs fixing, otherwise this is good to go if no one else has any concerns.

**Review by Shadowtail117 — APPROVED** [review]

**Pull request merged**
